package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@sap-devx/inquirer-gui",
   "displayName": "Inquirer GUI",
-  "version": "0.2.0",
+  "version": "0.3.0",
```

Comment:
0.2.1

Reply:
also the example app should be tested and consume version 0.2.1
https://github.com/SAP/inquirer-gui/blob/checkbox_add_max-height-slider/sample-app/package.json#L19

Reply:
will be done in a separate PR as first we need to publish the inquirer-gui to npm before consuming it. no?